src/main/features/response/views/check-and-send.njk

```diff
@@ -15,38 +15,34 @@
     {{ csrfProtection(csrf) }}
 
     {{ tableStart('yourDetails', 'Your details') }}
-    {{
```

Comment:
changes like this are big conflicts risk, can you undo and just apply your changes on top?